src/css/components/files/_live-search-data.scss

```diff
@@ -0,0 +1,10 @@
+.live-search-data {
+  width: 100%;
+  display: block;
+  height: 127px;
+  overflow-y: scroll;
+  overflow-x:hidden;
+  z-index: +10;
```

Comment:
Since this does not have a non-static position - just by looking at this ruleset - z-index will not have any affect. Remove if it is not making any difference.